### DISCUSSION TITLE
PostgREST + Auth "Unable to connect"

### DISCUSSION THREAD

**Discussion by jordiup** [supabase/supabase#20893] [Questions] [answered]
I'm currently receiving 529 and 500 errors for PostgREST + Auth 😭

This has been affecting my production project for about 8 hours, would love some help!

![image](https://github.com/supabase/supabase/assets/21688404/370df5ac-786f-4d19-bb56-a6902b4c504e)

![image](https://github.com/supabase/supabase/assets/21688404/4df841f9-a66f-476f-ad38-061c7ed46c83)

<img width="259" alt="image" src="https://github.com/supabase/supabase/assets/21688404/5dd54f1a-bd65-4ba1-9ec8-762d0059b0cb">

**Comment by jordiup** [comment, marked as answer]
How i fixed it:

<img width="568" alt="image" src="https://github.com/supabase/supabase/assets/21688404/eab47238-b21b-4485-863d-2c7470cd49a7">

<img width="267" alt="image" src="https://github.com/supabase/supabase/assets/21688404/00ad06a0-dfdc-46e3-843c-c091162d094b">

**Reply by trancethehuman** [reply]
I tried restarting but it still freezes
